lib/ui/window/platform_configuration.cc

```diff
@@ -77,12 +76,34 @@ void PlatformConfiguration::DidCreateIsolate() {
   report_timings_.Set(tonic::DartState::Current(),
                       Dart_GetField(library, tonic::ToDart("_reportTimings")));
 
-  // TODO(loicsharma): This should only be created if the embedder enables the
-  // implicit view.
-  // See: https://github.com/flutter/flutter/issues/120306
-  windows_.emplace(kImplicitViewId,
-                   std::make_unique<Window>(
-                       kImplicitViewId, ViewportMetrics{1.0, 0.0, 0.0, -1, 0}));
+  library_.Set(tonic::DartState::Current(), library);
+}
+
+void PlatformConfiguration::AddView(int64_t view_id,
+                                    const ViewportMetrics& view_metrics) {
+  auto [window_iterator, insertion_happened] = windows_.emplace(
+      view_id, std::make_unique<Window>(library_, view_id, view_metrics));
+  FML_DCHECK(insertion_happened);
+  // Make the new window send an AddView message to Dart.
+  window_iterator->second->AddView();
+}
+
+void PlatformConfiguration::RemoveView(int64_t view_id) {
+  FML_DCHECK(view_id != kFlutterImplicitViewId)
+      << "The implicit view #" << view_id << " should never be removed.";
+  size_t erased_elements = windows_.erase(view_id);
+  FML_DCHECK(erased_elements != 0) << "View #" << view_id << " doesn't exist.";
+  (void)erased_elements;
```

Comment:
What's the `(void)` cast for?

Reply:
This is to avoid a lint error that says `erased_elements` is unused in prod. It is also used in places such as `flow/frame_timings.cc`.

Reply:
Ah gotcha. I'd consider using `FML_DCHECK(windows_.find(view_id) != windows_.end())` instead, but I don't feel strongly. Feel free to keep as is